File: CRUD/PHP/Symfony3 V2/Symfony3/src/Sistema/Bundle/Controller/HomeController.php
<?php

// Written By Ismael Heredia in the year 2017

namespace Sistema\Bundle\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\Controller;

class HomeController extends Controller
{
    public function indexAction()
    {
        return $this->render('SistemaBundle:Home:index.html.twig');
    }
}

File: CRUD/C#/Aplicacion Web con MVC - LinQ/sistema/sistema/Controllers/HomeController.cs
// Written By Ismael Heredia in the year 2016

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace sistema.Controllers
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/

        public ActionResult Index()
        {

            if (Request.Cookies["user_login"] != null)
            {
                Conexiones.Conexion conexion_now = new Conexiones.Conexion();

                if (conexion_now.valid_cookie(Request.Cookies["user_login"].Value))
                {
                    return RedirectToAction("Index", "Administracion");
                }
                else
                {
                    return RedirectToAction("LogOn", "Login");
                }

            }
            else
            {
                return RedirectToAction("LogOn", "Login");
            }

        }

    }
}
